### PR TITLE
blog: add blog on SBOMs with wasmCloud

### PR DESCRIPTION
**Pull request by ericgregory**
Add blog on generating and scanning SBOMs for wasmCloud projects.

### CODE REVIEW

File: blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+title: "Secure your wasmCloud supply chain with Software Bills of Materials (SBOMs)"
+authors: [ericgregory]
+slug: 2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms
+image: './images/wasmcloud-sboms.webp'
+date: "2024-11-14"
+description: "In this blog, we'll explore how to generate SBOMs for wasmCloud projects using common open source tools syft and grype."
+
+---
+
+![Secure your wasmCloud supply chain with Software Bills of Materials (SBOMs)](./images/wasmcloud-sboms.webp)
```
Comment (joonas) [inline blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx]:
```suggestion
![Secure your wasmCloud supply chain with Software Bill of Materials (SBOMs)](./images/wasmcloud-sboms.webp)
```

File: blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Software supply chain security is critical for enterprises, and **Software Bills of Materials (SBOMs)** are an essential element of every organization's security framework.
```
Comment (joonas) [inline blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx]:
```suggestion
Software supply chain security is critical for enterprises, and **Software Bill of Materials (SBOMs)** are an essential element of every organization's security framework. 
```

**Review by joonas — APPROVED** [review]

File: blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx [hunk 3]
```diff
@@ -0,0 +1,91 @@
+---
+title: "Secure your wasmCloud supply chain with Software Bills of Materials (SBOMs)"
```
Comment (joonas) [inline blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx]:
```suggestion
title: "Secure your wasmCloud supply chain with Software Bill of Materials (SBOMs)"
```

Comment (ericgregory) [reply blog/2024-11-14-secure-your-wasmcloud-supply-chain-with-sboms/index.mdx]: I believe the original is grammatically correct but probably SEOishly suboptimal (and you never really see people use the plural); I'll adjust to use the singular "Software Bill of Materials" throughout!

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**